bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -818,6 +818,25 @@ spec:
            # REMOVEME once OVN path for control socket is fixed (right now uses /var/run/openvswitch)
           ln -sf $OVN_NB_DAEMON /var/run/ovn/ || true
 
+          # Check both nbServer and sbServer to make sure both are active before starting ovnkube-master
+          retries=0
+          while [ "$(ovn-appctl -t /var/run/ovn/ovnnb_db.ctl  ovsdb-server/sync-status | awk -F': ' '{print $2}')" != "active" ]; do
+             (( retries += 1 ))
+            if [[ "${retries}" -gt 10 ]]; then
+              echo "Northbound DB Server status not active for longtime. Continuing..."
+            fi
+            sleep 0.001
```

Comment:
@msherif1234 latency isn't critical at pod startup here since we only do it once in a while (hopefully!); I'd say 0.1 is a better balance between latency and hammering the db server.

I think this is OK, however keep in mind that this is checking the *local* DBs but ovnkube master is given the IPs of all databases and picks one from that list; it may not be the local DB. Also, just because the local DB is serving on its unix socket doesn't mean it's serving on TCP/SSL yet; that's a postStart for the DB pods:

https://github.com/openshift/cluster-network-operator/blob/596044d42085aa3b62b2571fa046de77600f055b/bindata/network/ovn-kubernetes/ovnkube-master.yaml#L262

What I'd do here instead is to use `ovn-nbctl find SSL` and `ovn-sbctl find SSL` along with the --db and SSL args to ensure that we can talk to at least one DB over SSL before declaring them "up" from onvkube-master's perspective.

Reply:
I thought you said it takes 8s between master starting and db starting? The math won't have the desired effect?

Reply:
that was what the log showed will make it 10 iteration 1sec apart